R/MakeDistMat.R

```diff
@@ -6,6 +6,15 @@
 #' takes in two vectors of equal length and produces a scalar
 #' @return Distance matrix with rows/columns named appropriately
 MakeDistMat = function(intensity_mat, dist_metric, symmetric=T){
+  # If there is only one cell type, return 1 x 1 matrix with NA entry
+  if(ncol(intensity_mat) == 1){
+    return(
+      matrix(NA, nrow = 1, ncol = 1) %>%
+        set_rownames(colnames(intensity_mat)) %>%
```

Comment:
can you make the `magrittr` namespace explicit here? as in, write magrittr::set_rownames?